app/components/x-toggle.js

```diff
@@ -5,8 +5,6 @@ const config = ENV['ember-cli-toggle'] || {};
 
 export default XToggle.extend({
   theme: config.defaultTheme || 'default',
-  offLabel: config.defaultOffLabel || 'Off',
-  onLabel: config.defaultOnLabel || 'On',
```

Comment:
I did default it to "Off:off" and "On:on" but we can change this back if you prefer